packages/unchained-client/src/types.ts

```diff
@@ -54,24 +54,13 @@ export enum TransferType {
   Contract = 'Contract'
 }
 
-export enum TxParser {
-  Cosmos = 'cosmos',
-  Yearn = 'yearn',
-  UniV2 = 'uniV2',
-  ZRX = 'zrx',
-  Thor = 'thor',
-  Foxy = 'foxy',
-  WETH = 'weth',
-  CowSwap = 'cowswap'
-}
-
 export interface BaseTxMetadata {
   method?: string
   parser: string
 }
 
 export interface StandardTxMetadata extends BaseTxMetadata {
-  parser: TxParser
+  parser: 'cosmos' | 'yearn' | 'uniV2' | 'zrx' | 'thor' | 'foxy' | 'weth' | 'cowswap'
```

Comment:
Removing this makes me a little sad. Would we still get the Intellisense improvements you are after if we kept the enum, but renamed it to `TxParsers`, and then created a type that is the string union of the enum values?

```ts
export enum TxParsers {
  Cosmos = 'cosmos',
  Yearn = 'yearn',
  UniV2 = 'uniV2',
  ZRX = 'zrx',
  Thor = 'thor',
  Foxy = 'foxy',
  WETH = 'weth',
  CowSwap = 'cowswap'
}

export type TxParser = keyof typeof TxParsers

export interface StandardTxMetadata extends BaseTxMetadata {
  parser: TxParser
}
```

Then we can keep using the enum in the tests and such instead of the hardcoded strings.

Reply:
That works, best of both worlds!
<img width="690" alt="image" src="https://user-images.githubusercontent.com/17035424/183087106-0aea63c1-f7de-42d9-9ce0-57e9e41793ec.png">

Reply:
Done in 042f8192406cbfb9697fa83c00f293c44831608f

Reply:
Perfect.